src/core/lib/security/security_connector/security_connector.h

```diff
@@ -106,6 +107,10 @@ grpc_security_connector* grpc_security_connector_from_arg(const grpc_arg* arg);
 grpc_security_connector* grpc_security_connector_find_in_args(
     const grpc_channel_args* args);
 
+/* Util to set the interested_parties whose ownership is not transferred. */
+void grpc_security_connector_set_interested_parties(
```

Comment:
This seems ugly.  Instead, why not simply add an interested_parties argument to `grpc_channel_security_connector_add_handshakers()` and `grpc_server_security_connector_add_handshakers()`?